src/api-reference/direct-connects/ground-transportation/post-reservation-sell.markdown

```diff
@@ -429,22 +445,22 @@ The response will include a **CC_LimoSellReply** parent element, with the follow
 |OptionalExtraStopCharge	|N|	|The charge for any additional stops.|
 |OptionalExtraTimeCharge	|N|	|The charge for each additional hour.|
 
-#### Vehicle elements
+#### <a name="vehicle"></a>Vehicle Elements
```

Comment:
Change this anchor to vehicle-schema. The browser is pointing to a previous vehicle heading, which it does automatically. In order to force the browser to point to this heading, we need the anchor to be a value that is not appearing in any other headers. Let me know if you have any questions.